### PR TITLE
Add filter on result on jury submissions page.

### PR DESCRIPTION
**Pull request by nickygerritsen**
As requested by @RagnarGrootKoerkamp .

Screenshot:
image

### CODE REVIEW

File: webapp/templates/jury/partials/submission_list.html.twig [hunk 1]
```diff
@@ -72,11 +72,27 @@
...
+                {% endif %}
+            {% elseif submission.judgings.first is empty or submission.judgings.first.result is empty %}
+                {% if submission.judgehost %}
+                    {% set result = 'judging' %}
+                {% else %}
+                    {% set result = 'queued' %}
+                {% endif %}
+            {% else %}
+                {% set result = submission.judgings.first.result %}
+            {% endif %}
```
Comment (ubergeek42) [inline webapp/templates/jury/partials/submission_list.html.twig:88]: this is ugly, but I don't see an obvious way to make it clearer

**Review by ubergeek42 — APPROVED** [review]
nice! :tada:

File: webapp/templates/jury/partials/submission_list.html.twig (hunk 1 above)
Comment (nickygerritsen) [reply webapp/templates/jury/partials/submission_list.html.twig:88]: Yes I had the same issue.... We could put this in a twig function and reuse it? But then too-late is hard

**Comment by thijskh** [comment]
Instead of keep adding text boxes, would it be difficult to make it one box?

Filter on: [ [Language C] [Result correct] ]

**Comment by nickygerritsen** [comment]
> Instead of keep adding text boxes, would it be difficult to make it one box?
> 
> Filter on: [ [Language C] [Result correct] ]

I wonder if that is not less user friendly? Probably technically it's possible but non-trivial (since we use 3 different AJAX requests for the top 3 boxes and none for the bottom one).

**Comment by RagnarGrootKoerkamp** [comment]
Gitlab does this, and I don't like it. I much prefer separate boxes.

File: webapp/templates/jury/partials/submission_list.html.twig (hunk 1 above)
Comment (thijskh) [reply webapp/templates/jury/partials/submission_list.html.twig:88]: Maybe we should not be doing this in twig in the first place?
Something for the controller to determine what the 'verdict' is,including juding and queued. And also add a too-late flag as well that twig can act on?

Comment (nickygerritsen) [reply webapp/templates/jury/partials/submission_list.html.twig:88]: Yes this might be better. I will try it

Comment (nickygerritsen) [reply webapp/templates/jury/partials/submission_list.html.twig]: Gave it a try

File: webapp/src/Twig/TwigExtension.php [hunk 2]
```diff
@@ -549,6 +550,53 @@ public function printValidJuryResult($result): string
...
 
+    /**
+     * Print the given submission for the jury, assuming it is valid
+     *
+     * @param Submission $submission
+     * @param bool       $forDisplay
+     *
+     * @return string
+     */
+    public function printValidJurySubmissionResult(Submission $submission, bool $forDisplay = true)
```
Comment (meisterT) [inline webapp/src/Twig/TwigExtension.php:561]: nit: add `: string`

**Review by meisterT — APPROVED** [review]

**Pull request merged**